Assets/Enemy/EnemyData.cs

```diff
@@ -2,6 +2,7 @@
 using System;
 using System.Collections.Generic;
 using System.Xml.Serialization;
+using UnityEngine;
```

Comment:
Do you need this using declaration?

Reply:
I didn't add that line, visual studio must have.  Also when I skimmed the changes I thought it was deleting the using.